### PR TITLE
BUG: fix python command-block to no-exec

### PR DESCRIPTION
**Pull request by ebolyen**
The command block directive is trying to run a shell command. Just add the `:no-exec:` directive argument to prevent that issue.

(Also fixed indentation)

### CODE REVIEW

File: source/sphinx_extensions/plugin_directory/templates/action.rst [hunk 1]
```diff
@@ -39,8 +39,9 @@
        <h4>Import:</h4>
 
 .. command-block::
+   :no-exec:
 
-       from {{import_path}} import {{action_api_name}}
+   from {{import_path}} import {{action_api_name}}
```
Comment (thermokarst) [inline source/sphinx_extensions/plugin_directory/templates/action.rst]: Can you drop the blank line below the import? @ChrisKeefe & I were just looking at this - it adds a newline to the copy-to-clipboard button.

Comment (ebolyen) [reply source/sphinx_extensions/plugin_directory/templates/action.rst]: Removing the newline does not seem to fix the issue (also all of our directives are formatted with an extra newline to separate the directive from the rest of the content).

Comment (thermokarst) [reply source/sphinx_extensions/plugin_directory/templates/action.rst]: Bummer.

**Comment by ChrisKeefe** [comment]
Looks good to me, Evan. Thanks!

**Pull request merged**